src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocInlineDefault.java

```diff
@@ -52,10 +52,29 @@ void foo12() throws Exception {}
     /** {@summary An especially short bit of Javadoc.} */
     void foo13() {} // ok
 
+    // violation below
+    /**
+     * {@return}
+     */
+    int returnNothing() {
+        return 0;
+    }
+
     /**
      * {@summary Some Javadoc.}
      */
     public static final byte NUL = 0; // ok
+
+    /**
+     * {@return nothing, this is a field}
```

Comment:
`nothing, this is a field, note that no period is ok as it is set by javadoc tool`

to tell next engineer a nuance

Reply:
I don't think this is the right place to add this nuance? I added it to a "correct" variant.